src/cryptonote_core/miner.h

```diff
@@ -82,6 +82,7 @@ namespace cryptonote
     void pause();
     void resume();
     void do_print_hashrate(bool do_hr);
+    uint64_t get_block_reward() const { return m_block_reward; }
```

Comment:
It was reverting a change in the commit that removed background mining because this call is still needed in the RPC code.  (Or at least would be -- looks like it's commented out right now, will fix).